haystack/pipelines/base.py

```diff
@@ -1844,31 +1850,31 @@ def load_from_yaml(
 
         Here's a sample configuration:
 
-            ```yaml
-            |   version: '1.9.0'
-            |
-            |    components:    # define all the building-blocks for Pipeline
-            |    - name: MyReader       # custom-name for the component; helpful for visualization & debugging
-            |      type: FARMReader    # Haystack Class name for the component
-            |      params:
-            |        model_name_or_path: deepset/roberta-base-squad2
-            |    - name: MyESRetriever
-            |      type: BM25Retriever
-            |      params:
-            |        document_store: MyDocumentStore    # params can reference other components defined in the YAML
-            |    - name: MyDocumentStore
-            |      type: ElasticsearchDocumentStore
-            |      params:
-            |        index: haystack_test
-            |
-            |    pipelines:    # multiple Pipelines can be defined using the components from above
-            |    - name: my_query_pipeline    # a simple extractive-qa Pipeline
-            |      nodes:
-            |      - name: MyESRetriever
-            |        inputs: [Query]
-            |      - name: MyReader
-            |        inputs: [MyESRetriever]
-            ```
+           ```yaml
+           version: '1.9.0'
+
+            components:    # define all the building-blocks for Pipeline
+            - name: MyReader       # custom-name for the component; helpful for visualization & debugging
+              type: FARMReader    # Haystack Class name for the component
+              params:
+                model_name_or_path: deepset/roberta-base-squad2
+            - name: MyESRetriever
+              type: BM25Retriever
+              params:
+                document_store: MyDocumentStore    # params can reference other components defined in the YAML
+            - name: MyDocumentStore
+              type: ElasticsearchDocumentStore
+              params:
+                index: haystack_test
+
+            pipelines:    # multiple Pipelines can be defined using the components from above
+            - name: my_query_pipeline    # a simple extractive-qa Pipeline
+              nodes:
+              - name: MyESRetriever
+                inputs: [Query]
+              - name: MyReader
+                inputs: [MyESRetriever]
+           ```
```

Comment:
Same
```suggestion
           ```yaml
           version: '1.9.0'

            components:    # define all the building-blocks for Pipeline
            - name: MyReader       # custom-name for the component; helpful for visualization & debugging
              type: FARMReader    # Haystack Class name for the component
              params:
                model_name_or_path: deepset/roberta-base-squad2
            - name: MyRetriever
              type: BM25Retriever
              params:
                document_store: MyDocumentStore    # params can reference other components defined in the YAML
            - name: MyDocumentStore
              type: ElasticsearchDocumentStore
              params:
                index: haystack_test

            pipelines:    # multiple Pipelines can be defined using the components from above
            - name: my_query_pipeline    # a simple extractive-qa Pipeline
              nodes:
              - name: MyRetriever
                inputs: [Query]
              - name: MyReader
                inputs: [MyRetriever]
           ```
```